README.md

```diff
@@ -53,13 +53,39 @@ npx hardhat test
 8. Run the deployment script to deploy the latest contracts to the testnet.
 
 ```
-npx hardhat deploy --network rinkeby
+npx hardhat deploy --network rinkeby --tag Deployment
 ```
 
 ## Upgrading
 
 Upgrading contracts that are owned by a multisig should be proposed using [Defender](https://docs.openzeppelin.com/defender/guide-upgrades)
 
+1. Sign up for a Defender account and generate an API key.
+
+2. Export the API key and secret key into your local environment.
+
+```
+export DEFENDER_TEAM_API_KEY=<API_KEY>
+export DEFENDER_TEAM_API_SECRET_KEY=<SECRET_KEY>
+```
+
+3. Export the addresses of the token proxy contract to be upgraded and the owning Gnosis multisig address.
+
+```
+export ARROW_TOKEN_PROXY_CONTRACT=<TOKEN_ADDRESS>
+export ARROW_TOKEN_MUTISIG=<MULTISIG_ADDRESS>
+```
+
+4. Run the upgrade script to generate an upgrade proposal in Defender.
+
+```
+npx hardhat deploy --tags Upgrade --network rinkeby
```

Comment:
```suggestion
npx hardhat deploy --network rinkeby --tags Upgrade 
```

The order of these args should be consistent.

Reply:
If it bothers you. 😉